events/EiffelSourceChangeCreatedEvent/simple.json

```diff
@@ -0,0 +1,45 @@
+{
+  "meta": {
+    "type": "EiffelSourceChangeCreatedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "domainId": "example.domain",
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0"
+  },
+  "data": {
+    "gitIdentifier": {
+      "commitId": "fd090b60a4aedc5161da9c035a49b14a319829b4",
+      "branch": "myBranch",
+      "repoName": "myPrivateRepo",
+      "repoUri": "https://github.com/johndoe/myPrivateRepo.git"
+    },
+    "author": {
+      "name": "John Doe",
+      "email": "john.doe@company.com",
+      "id": "johndoe",
+      "group": "Team Gophers"
+    },
+    "change": [
```

Comment:
Change is not defined as an array. Do not see the reason why it should be. Remove []?